ee/secureenclaverunner/secureenclaverunner_test.go

```diff
@@ -215,4 +247,116 @@ func Test_secureEnclaveRunner(t *testing.T) {
 		// and public not called
 		require.Len(t, ser.uidPubKeyMap, 0)
 	})
+
+	t.Run("creates new key when existing not found in secure enclave", func(t *testing.T) {
```

Comment:
Just curious -- I noticed these tests use `t.Run` instead of separate test functions. Is that a pattern you think we might want to adopt more widely across launcher? I like how specific the test names are without having to be like `func TestFunc_someSpecificHappyOrUnhappyPath(t *testing.T)`.

Reply:
I just kind of tried it out and I really like it for a few reasons:

1. The table driven tests can get really complicated as the the code it's testing grows and more edge cases are found
2. Using VS Code I can click run on a single test, for the table driven tests I have to comment out the ones I don't want to run
3. Feels easier to add or remove things from the tests, with the table driven tests, you change one field in the struct and your updating n tests

I still think table driven tests have their place and are a better choice for smaller scoped areas of code with simple inputs, outputs and few error / success paths.